dev-docs/source/auth.rst

```diff
@@ -23,13 +23,14 @@ At every level, the ``auth`` field contains a string that is called a *scope*.
 A request's JWT must contain this scope in its ``scopes`` claim
 (see the ``authorization_django`` source code for details)
 to meet an ``auth`` field's requirements.
-The absence of an ``auth`` field marks a publicly available resource.
+The absence of a scope marks a publicly available resource.
 
-The presence of ``auth`` fields on multiple levels
-means that all listed scopes must be carried in the JWT.
-So, if the dataset has ``"auth": "FOO/BAR"`` and a field has ``"auth": "FOO/BAZ``,
-then access to the field is only available
-with a token that carries the ``FOO/BAR`` and ``FOO/BAZ`` scopes.
+Scopes at higher levels override scopes at lower levels.
```

Comment:
*higher levels of schema.
Otherwise it may be confusing people that scopes also have hierarchy, which they do not really.

Reply:
Good point.